website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,239 @@
+
+[FedRAMP](/using-unleash/compliance/fedramp): A U.S. government program that standardizes how federal agencies assess, authorize, and monitor cloud services. For organizations listing their software on the [FedRAMP](https://www.fedramp.gov/program-basics/) marketplace, Unleash's security controls are designed to meet these rigorous requirements.
+
+[SOC 2 Type II](/using-unleash/compliance/soc2): A cybersecurity framework that assesses how organizations protect customer data. Our architecture provides the necessary controls and audit trails to meet stringent SOC 2 requirements, ensuring your feature flag management processes are transparent, secure, and verifiable.
+
+[ISO 27001](/using-unleash/compliance/iso27001): A framework from the [International Organization for Standardization (ISO)](https://www.iso.org/standard/27001) for managing information security within an organization. Our security, audit, and change control capabilities align with ISO 27001 standards, providing a structured approach to managing sensitive information and protecting data integrity.
+
+Refer to our [security and compliance overview](/using-unleash/compliance/compliance-overview) for more details on how Unleash features measure up against these framework requirements.
+
+## Enable authentication controls with SSO and SCIM
+
+The first step to securely administering your feature flag system is to consider the authentication process. When you’re using Unleash, how should your team members access the platform UI? We have multiple ways for users to log into the platform, including traditional, password-based login. But when you’re considering the best approach to using feature flags in your development process, you need to consider how every access point to your projects, data, and development processes could pose a security risk for your organization. To demonstrate that your tools are configured with security and compliance in mind, we recommend you enable enterprise-grade authentication controls as your first line of defense.
+
+Your developers and other stakeholders need to securely access platforms used to build and ship software to production. At Unleash, we have a multi-layered authentication architecture that supports [single sign-on](/reference/sso) (SSO) and [SCIM](/reference/scim) protocols. These authentication controls are the de facto ways for you to integrate enterprise-grade security measures easily. Let’s take a closer look at each authentication option to use.
+
+### Use SSO authentication for feature flags
+
+To use single sign-on in Unleash, your users can authenticate themselves through OpenID Connect (OIDC) or SAML 2.0 protocols.
+
+We have integration options with enterprise identity providers like Okta, Microsoft Entra ID, and Keycloak. Connect these identity providers through either OpenID Connect or SAML 2.0 protocol. For a step-by-step guide, read our [how-to guide for single sign-on](/how-to/sso).
```

Comment:
Dunno if this changes how you want to frame this but we support anything and everything that either has SAML or OIDC, it's just that we have comprehensive docs for the options you list

Reply:
Thanks for mentioning this! To me, the docs implied these were the only identity providers we supported. So I will certainly update this to make it more clear what the options are / could be.

Reply:
Ooh I didn't know this 🤔 I'll send you an updated diagram